src/portfolio.js

```diff
@@ -119,12 +119,12 @@ const techStack = {
 // Your top 3 work experiences
 
 const workExperiences = {
-  viewExperiences: true, //Set it to true to show workExperiences Section
+  viewExperiences: false, //Set it to true to show workExperiences Section
```

Comment:
Why is this set to false?

Reply:
I was testing the component.
Enabled it though